### PR TITLE
Check for `--strict` along with `--noImplicitAny` for code fix

### PR DESCRIPTION
**Pull request by mhegazy**
Fixes https://github.com/Microsoft/TypeScript/issues/19378

### CODE REVIEW

**Comment by mhegazy** [comment]
error related to gulp-typescript version.

File: src/services/refactors/installTypesForPackage.ts [hunk 1]
```diff
@@ -12,7 +12,8 @@ namespace ts.refactor.installTypesForPackage {
     registerRefactor(installTypesForPackage);
 
     function getAvailableActions(context: RefactorContext): ApplicableRefactorInfo[] | undefined {
-        if (context.program.getCompilerOptions().noImplicitAny) {
+        const options = context.program.getCompilerOptions();
+        if (options.noImplicitAny || options.strict) {
```
Comment (weswigham) [inline src/services/refactors/installTypesForPackage.ts]: What about `strict: true` with `noImplicitAny: false`?

**Comment by mhegazy** [comment]
You are right. Will fix it

File: src/services/refactors/installTypesForPackage.ts (hunk 1 above)
Comment (mhegazy) [reply src/services/refactors/installTypesForPackage.ts]: fix up in https://github.com/Microsoft/TypeScript/pull/19427

**Pull request merged**